examples/non-networked-paraent.html

```diff
@@ -0,0 +1,188 @@
+    <button id="resetButton" onclick="restPlatformHeight()" class="button resetButton">Reset</button>
+
+    <a-scene
+      networked-scene="
+      room: basic;
+      debug: true;
+      adapter: wseasyrtc;
+    "
+    >
+      <a-assets>
+        <!-- Templates -->
+        <!-- Camera Rig / Player -->
+        <template id="rig-template">
+          <a-entity></a-entity>
+        </template>
+
+        <!-- Head / Avatar -->
+        <!--      a few spheres make a head + eyes + pupils    -->
+        <template id="avatar-template">
+          <a-entity class="avatar">
+            <!-- notice this child sphere, with class .head, has the random-color component; this modifies the material component's color property -->
+            <a-sphere class="head" scale="0.2 0.22 0.2" random-color></a-sphere>
+            <a-entity class="face" position="0 0.05 0">
+              <a-sphere class="eye" color="white" position="0.06 0.05 -0.16" scale="0.04 0.04 0.04">
+                <a-sphere class="pupil" color="black" position="0 0 -1" scale="0.2 0.2 0.2"></a-sphere>
+              </a-sphere>
+              <a-sphere class="eye" color="white" position="-0.06 0.05 -0.16" scale="0.04 0.04 0.04">
+                <a-sphere class="pupil" color="black" position="0 0 -1" scale="0.2 0.2 0.2"></a-sphere>
+              </a-sphere>
+            </a-entity>
+          </a-entity>
+        </template>
+        <!-- /Templates -->
+      </a-assets>
+
+      <a-entity environment="preset:starry;groundColor:#000000;"></a-entity>
+      <a-entity light="type:ambient;intensity:0.5"></a-entity>
+
+      <!-- Networked entities will be added as children of the platform -->
+      <a-box id="platform" color="tomato" depth="10" height=".5" width="10" position="0 10 0">
+        <!--   Here we declare only the local user's avatar, which we then broadcast to other users     -->
+        <!--   The 'spawn-in-circle' component will set the position and rotation of #rig;
+             because this entity also has the networked component, and position and rotation are tracked by default,
+             the changes made by spawn-in-circle will be kept in sync with other networked users.
+             Also note that by adding the networked component with a template reference, we generate that full template,
+             including all applicable child elements. However, because we don't need to see our own avatar, we use the
+             `attachTemplateToLocal:false` option. This makes our local copies invisible on our machine, but visible on everyone else's.
+      -->
+        <a-entity
+          id="rig"
+          movement-controls="fly:true;"
+          spawn-in-circle="radius:3"
+          networked="template:#rig-template; allowNonNetworkedParent:true;"
+        >
+          <!-- We add allowNonNetworkedParent because we want networked entities to end up as a child of platform entity not the root a-scene.
+          -->
```

Comment:
I like that we've added this documentation as comments. I would recommend also adding additional information visible in the UI of this example explaining what this example is and how to use it without needing to inspect the source code. See the "Persistent spheres" example which includes the header at the top as an example.